examples/balances.inf

```diff
@@ -1,27 +1,28 @@
 ! ----------------------------------------------------------------------------
-!   Balances 961216                 One of the standard Inform 6 example games
+!   Balances 160701                 One of the standard Inform 6 example games
 !
 !                                                             created: 25.9.94
 !                                                             updated: 6.10.94
 !                                                         modernised: 11.12.95
 !                                               translated to Inform 6: 8.5.96
 !                                                   minor bugs fixed: 16.12.96
+!                                 modernised again by Vince LaViano :  26.6.28
```

Comment:
This date is in the reverse format of the previous ones - maybe update them to to be YY.MM.DD? Also, it's only 2016, not 2026 ;)